mw-chat/internal/db/query/rooms.query.sql

```diff
@@ -6,9 +6,43 @@ RETURNING uuid;
 -- name: GetRoomsByUserUUID :many
 SELECT
     rooms.uuid,
-    rooms.type
+    rooms.name,
+    rooms.type,
+    users_rooms.is_room_blocked,
+    ARRAY(
+        SELECT
+            users_rooms.user_uuid
+        FROM users_rooms
+        WHERE users_rooms.room_uuid = rooms.uuid
+    )::UUID[] AS user_uuids,
+    ARRAY(
+            SELECT
+                users_rooms.role
+            FROM users_rooms
+            WHERE users_rooms.room_uuid = rooms.uuid
+    )::VARCHAR[] AS user_roles
```

Comment:
such code looks unsafe, does it possible  to declare core of the function somewere nearby?